src/utils/param-preview.utils.ts

```diff
@@ -11,10 +11,12 @@ import { tzToMutez } from './tezos.util';
 export const getParamPreview = (opParam: ParamsWithKind): ParamPreviewInterface => {
   if (opParam.kind === OpKind.DELEGATION) {
     // Delegate
-    return {
-      type: ParamPreviewTypeEnum.Delegate,
-      baker: opParam.delegate
-    };
+    if (isDefined(opParam.delegate)) {
```

Comment:
combine this condition with condition `opParam.kind === OpKind.DELEGATION` above.